forms/login.go

```diff
@@ -0,0 +1,33 @@
+package forms
+
+import (
+	"github.com/beego/beego/v2/core/validation"
+)
+
+type LoginForm struct {
```

Comment:
How about separating the forms for API and Web? To login, I belive there would be different field required for each.

Reply:
@olivierobert moved this form to `/api` on [82a88b0](https://github.com/carryall/go-google-scraper-challenge/pull/26/commits/82a88b0b5c15918a7daaa72cc10db8b495fb5cb9) and I'll work on another form for web login on [another PR](https://github.com/carryall/go-google-scraper-challenge/pull/31)